### PR TITLE
Feature/200 screening questions updates

### PR DESCRIPTION
**Pull request by lukaszdrynski**

### CODE REVIEW

File: app/routes/v2.js [hunk 1]
```diff
@@ -83,15 +73,44 @@ var router = express.Router();
...
-  router.post('/screening-questions/respondent-address', function (req, res) {
-    res.redirect('same-sex');
+    router.post('/screening-questions/marriage-cert', function (req, res) {
+        var haveMarriageCert = req.body.marriageCert;
+        var isInEnglish = req.body.isInEnglish;
+        req.session.certificate.haveMarriageCert = haveMarriageCert;
+        req.session.certificate.isInEnglish = isInEnglish;
+
+        if (haveMarriageCert === 'No') {
+            res.redirect('../exit/sq/no-certificate');
```
Comment (sophiedrake) [inline app/routes/v2.js]: sq, screening-questions would be better IMHO

File: app/views/v2/screening-questions/marriage-cert-translated.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <p>You must provide either a:
+    <ul class="list list-bullet">
+        <li><a href="https://www.gov.uk/certifying-a-document" target="_blank">certified</a> translation by a qualified
+            translator
+        </li>
+        <li>notarised translation (translated by a qualified translator, and then certified by a public notary)</li>
+    </ul>
+    </p>
+
+    <label for="translated-yes" class="block-label">
```
Comment (sophiedrake) [inline app/views/v2/screening-questions/marriage-cert-translated.html]: Should this check the session to see if its already been answered and check the correct one if it has

File: app/views/v2/screening-questions/marriage-cert.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+{% endblock %}
+
+{% block form %}
+
+<fieldset class="form-group inline" id="marriage-cert">
+
+    <p>This must be the original or a <a href="https://www.gov.uk/order-copy-birth-death-marriage-certificate"
+                                         target="_blank">certified copy</a></p>
+
+    <label for="cert-yes" class="block-label"
```
Comment (sophiedrake) [inline app/views/v2/screening-questions/marriage-cert.html]: Same with this view, check session for existing value

File: app/views/v2/screening-questions/marriage-type.html [hunk 4]
```diff
@@ -1,92 +1,26 @@
...
+        <input id="marriage" type="radio"
+               name="marriageType" value="marriage">
+        End a marriage
     </label>
 
-    <label for="cert-no" class="block-label">
-        <input id="cert-no" type="radio"
-               name="marriageCert" value="No">
-        No
+    <label for="same-sex-marriage" class="block-label">
```
Comment (sophiedrake) [inline app/views/v2/screening-questions/marriage-type.html]: Think this needs to check for a value and 'check' as well

File: app/views/v2/screening-questions/respondent-address.html [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
 
 <fieldset class="form-group inline">
     <label class="form-label">
-        The court will need this so it can send them your divorce application.
+        The court needs to send them a copy of this application form.
     </label>
+    <label class="form-label">You will need the full address. This can be based in the UK or abroad.</label>
     <label for="respondentAddress-yes" class="block-label">
         <input id="respondentAddress-yes" type="radio"
                name="respondentAddress" value="Yes">
```
Comment (sophiedrake) [inline app/views/v2/screening-questions/respondent-address.html]: Check session here too

File: app/routes/v2.js (hunk 1 above)
Comment (sophiedrake) [inline app/routes/v2.js]: Should be `screening-questions`

File: app/views/v2/exit/screening-questions/no-certificate-translated.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    <span class="form-label">Please obtain either a:</span>
+    <ul class="list list-bullet">
+        <li><a href="https://www.gov.uk/certifying-a-document" target="_blank">certified</a> translation by a qualified
+            translator
+        </li>
+        <li>notarised translation (translated by a qualified translator, and then certified by a public notary)</li>
+    </ul>
+    <p>You can apply again once you have these documents. You will need to scan or take a picture of the documents above
+        and upload them as part of this application.</p>
+</div>
```
Comment (sophiedrake) [inline app/views/v2/exit/screening-questions/no-certificate-translated.html]: Does this need an `Exit Application` button as per screens?

**Comment by sophiedrake** [comment]
Does `app/views/exit.html` need to be updated with an `Exit application` button or similar as per screens?

**Comment by lukaszdrynski** [comment]
existence of `Exit application` button on exit screens is not 100% confirmed yet, other changes are implemented

**Comment by sophiedrake** [comment]
+1

**Pull request merged**